packages/react-devtools-shell/src/e2e/apps/ListApp.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import * as React from 'react';
```

Comment:
No new JSX transform? 🥺

Reply:
We should update the shell for this! Wanna do it in a follow up PR?